code/game/jobs/jobs.dm

```diff
@@ -118,6 +118,13 @@ var/list/security_positions = list(
 )
 
 
+var/list/planet_positions = list(
+	"Explorer",
+	"Pilot",
+	"Search and Rescue"

```

Comment:
These are in different departments on ours. Explorer (and subjobs we made on ours) are part of R&D, Pilot is civ, and SAR is med.